setup.py

```diff
@@ -454,9 +468,7 @@ def _read_requirements(filename: str) -> List[str]:
     python_requires=">=3.8",
     install_requires=get_requirements(),
     ext_modules=ext_modules,
-    extras_require={
-        "tensorizer": ["tensorizer>=2.9.0"],
-    },
-    cmdclass={"build_ext": cmake_build_ext} if _build_custom_ops() else {},
+    extras_require=get_extra_requirements(),
+    cmdclass={"build_ext": cmake_build_ext} if not _is_neuron() else {},
```

Comment:
```suggestion
    cmdclass={"build_ext": cmake_build_ext} if _build_custom_ops() else {},
```

https://github.com/vllm-project/vllm/blob/main/setup.py#L460